shared/properties-view/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+cmake_minimum_required(VERSION 3.16...3.25)
+
+find_package(Qt6 REQUIRED Core Widgets)
+
+if(NOT TARGET OBS::qt-wrappers)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/shared/qt/wrappers" "${CMAKE_BINARY_DIR}/shared/qt/wrappers")
+endif()
+
+if(NOT TARGET OBS::qt-plain-text-edit)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/shared/qt/plain-text-edit" "${CMAKE_BINARY_DIR}/shared/qt/plain-text-edit")
+endif()
+
+if(NOT TARGET OBS::qt-vertical-scroll-area)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/shared/qt/vertical-scroll-area"
+                   "${CMAKE_BINARY_DIR}/shared/qt/vertical-scroll-area")
+endif()
+
+if(NOT TARGET OBS::qt-slider-ignorewheel)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/shared/qt/slider-ignorewheel"
+                   "${CMAKE_BINARY_DIR}/shared/qt/slider-ignorewheel")
+endif()
+
+add_library(properties-view INTERFACE)
```

Comment:
It might be more beneficial to make this (and the other qt-based) shared modules `OBJECT` libraries so they only have to be compiled once. `INTERFACE` type libraries just introduce the source and header files to the target, extending/duplicating compilation effort.

Reply:
Just attempted this and it's not possible without a rewrite of the affected classes - they are all too intertwined to be compiled as independent units.

Reply:
Unfortunately, Qt needs to use at least MOC on those files (creating another object file) and this cause trouble in CMake (mising symbol from MOC objects) so I will not dig deeper in the scope of this PR.